browser/resources/settings/brave_default_extensions_page/brave_default_extensions_page.js

```diff
@@ -2,6 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import './change_ipfs_gateway_dialog.js';
```

Comment:
Unfortunately since `brave_default_extensions_page` still uses html imports and has not yet been migrated to `html_to_js`, we should surround this in a comment, like:

```
// clang-format off
// #import './change_ipfs_gateway_dialog.js'
// clang-format on
```

This is how chromium does it anyway for these not-yet-migrated modules.

Reply:
Not sure why but this does not seem to imported when wrapped like this, current import works fine in component build.
Building debug build at the moment to see if it works well in debug too.

Reply:
@petemill I tried in Debug, Release, and Component builds, all works fine.